src/core/ext/xds/xds_client.cc

```diff
@@ -261,7 +266,7 @@ class XdsClient::ChannelState::AdsCallState
         ResourceState& state = authority_state.resource_map[type_][name_.key];
         state.meta.client_status = XdsApi::ResourceMetadata::DOES_NOT_EXIST;
         ads_calld_->xds_client()->NotifyWatchersOnResourceDoesNotExist(
-            state.watchers, XdsApi::ReadDelayHandle::NoWait());
+            state.watchers, XdsClient::ReadDelayHandle::NoWait());
```

Comment:
I don't think we need the `XdsClient::` prefix, since we're already inside that class.

Same thing throughout.